Realm/Realm/Handles/SessionHandle.cs

```diff
@@ -96,12 +110,19 @@ public static void Initialize()
             NativeMethods.SessionErrorCallback error = HandleSessionError;
             NativeMethods.SessionProgressCallback progress = HandleSessionProgress;
             NativeMethods.SessionWaitCallback wait = HandleSessionWaitCallback;
+            NativeMethods.SessionPropertyChangedCallback propertyChanged = HandleSessionPropertyChangedCallback;
 
             GCHandle.Alloc(error);
             GCHandle.Alloc(progress);
             GCHandle.Alloc(wait);
+            GCHandle.Alloc(propertyChanged);
 
-            NativeMethods.install_syncsession_callbacks(error, progress, wait);
+            NativeMethods.install_syncsession_callbacks(error, progress, wait, propertyChanged);
+        }
+
+        ~SessionHandle()
```

Comment:
This is not necessary - the SafeHandle finalizer should always call `ReleaseHandle`.

Reply:
I still see the need to dispose of the notification token. Are you suggesting to put that call (`UnsubscribeNotifications`) in `ReleaseHandle`?